localization/README.md

```diff
@@ -0,0 +1,87 @@
+# Localization
+
+The extension has 2 main ways to localize the strings in the UI.
+
+## 1. Static contributions in `package.nls.json`
+
+The extension uses the `package.nls.json` file to localize the strings the contributions in the `package.json` file. The `package.nls.json` file is a simple JSON file that maps the keys to localized strings.
+
+Example of `package.nls.json`
+
+```json
+{
+  "mssql.key1": "Localized string 1",
+  "mssql.key2": "Localized string 2"
+}
+```
+
+Using in `package.json` to contribute to vscode extensibility points. For example: commands
+
+```json
+{
+  "commands": [
+    {
+      "command": "mssql.testCommand",
+      "title": "%mssql.key1%",
+      "icon": "$(debug-start)"
+    }
+  ]
+}
+```
+
+## 2. Dynamic contributions in `*.ts` and `*.tsx` files
```

Comment:
There's also the mostly-dynamic method of defining it in the locConstants file, right?  Can you add a blub here mentioning that file as well, and providing a recommendation of when to use one over the other?  (shared strings vs. one-time-use strings, converted "legacy" localized strings vs. new localized strings)

Reply:
It doesn't matter where we define the strings now as long as the strings are in src folder in 'ts' or 'tsx' files. I am just using locConstants as a way to reduce the changes for the PR.